src/index.html

```diff
@@ -11,7 +11,214 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" class="form">
+      <fieldset>
+        <legend>Personal information</legend>
+        <div>
+          <label for="surname">Surnmae: </label>
+
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="name">Name: </label>
+
+          <input
+            type="text"
+            name="name"
+            id="name"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+
+          <input
+            type="number"
+            name="age"
+            id="age"
+            autocomplete="off"
+            value="12"
+            min="1"
+            max="100"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="date-birth">Full date of birth: </label>
+
+          <input
+            type="date"
+            name="date-birth"
+            id="date-birth"
+            autocomplete="off"
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="agreement">I accept the term of the agreement </label>
+
+          <input
+            type="checkbox"
+            name="agreement"
+            id="agreement"
+            autocomplete="off"
+            required
+          >
+        </div>
+      </fieldset>
+
+
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+        <div>
+          <label for="email">E-mail: </label>
+
+          <input
+            type="email"
+            name="email"
+            id="email"
+            autocomplete="off"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="password">Password: </label>
+
+          <input
+            type="password"
+            name="password"
+            id="password"
+            autocomplete="off"
+            required
+            minlength="5"
+            maxlength="10"
+          >
+        </div>
+      </fieldset>
+
+
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+        <div>
+          <label for="yes">Do you love cats?</label>
+
+          <input
+            type="radio"
+            name="choice"
```

Comment:
Lets make this name more descriptive so it is easier to use (in scenartio when form is being send)
```suggestion
            name="lovesCats"
```